thirdparty/lua-ljsqlite3/init.lua

```diff
@@ -73,34 +73,34 @@ do
   local types = { "INTEGER", "FLOAT", "TEXT", "BLOB", "NULL" } -- From 1 to 5.
 
   local opens = {
-    READONLY =        0x00000001;
```

Comment:
What about the third-party thing?

Reply:
That was just me trying to get it to work under Busted (it still doesn't ;p).